This patch removes the invocation of get_autostart() and
set_autostart(active). So gtk-redshift won't autostart (see #619699)
Index: redshift/src/redshift-gtk/statusicon.py
===================================================================
--- redshift.orig/src/redshift-gtk/statusicon.py	2014-08-05 15:03:21.262084904 +0530
+++ redshift/src/redshift-gtk/statusicon.py	2014-08-05 15:04:16.034652291 +0530
@@ -94,18 +94,6 @@
         suspend_menu_item.set_submenu(suspend_menu)
         self.status_menu.append(suspend_menu_item)
 
-        # Add autostart option
-        autostart_item = Gtk.CheckMenuItem.new_with_label(_('Autostart'))
-        try:
-            autostart_item.set_active(utils.get_autostart())
-        except IOError as strerror:
-            print(strerror)
-            autostart_item.set_property('sensitive', False)
-        else:
-            autostart_item.connect('toggled', self.autostart_cb)
-        finally:
-            self.status_menu.append(autostart_item)
-
         # Add info action
         info_item = Gtk.MenuItem.new_with_label(_('Info'))
         info_item.connect('activate', self.show_info_cb)
